app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -3,7 +3,13 @@ module VerifyProfileConcern
 
   def url_for_pending_profile_reason
     return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
-    return idv_in_person_ready_to_verify_url if current_user.in_person_pending_profile?
```

Comment:
I'm curious if it would be enough to just remove this line altogether, since my assumption is that the IdV request would kick the user into the IdV flow, and the IdV flow will automatically redirect to the screen anyways when an IPP enrollment is pending:

https://github.com/18F/identity-idp/blob/d92c6d87c05a09cc3fe3e59e715ad3a86253426a/app/controllers/openid_connect/authorization_controller.rb#L28

https://github.com/18F/identity-idp/blob/d92c6d87c05a09cc3fe3e59e715ad3a86253426a/app/controllers/idv_controller.rb#L32

https://github.com/18F/identity-idp/blob/d92c6d87c05a09cc3fe3e59e715ad3a86253426a/app/controllers/idv/welcome_controller.rb#L3

https://github.com/18F/identity-idp/blob/d92c6d87c05a09cc3fe3e59e715ad3a86253426a/app/controllers/concerns/idv_step_concern.rb#L13

https://github.com/18F/identity-idp/blob/d92c6d87c05a09cc3fe3e59e715ad3a86253426a/app/controllers/concerns/idv_step_concern.rb#L22-L25

Reply:
We could make it so we dont fall into that flow if its a pending profile for in person, a question I had is do we want the same level of blocking for GPO as well?

Reply:
got ya.